basic-android-kotlin-compose-training-mars-photos-starter/app/src/main/java/com/example/marsphotos/ui/screens/realEstateScreen/RealEstateViewModel.kt

```diff
@@ -26,6 +27,17 @@ class RealEstateViewModel(
         getRealEstatePrice()
     }
 
+
+    var selectedPosition: Int by mutableIntStateOf(-1)
+
+    fun savePosition(index: Int) {
+        selectedPosition = index
+    }
+
+    fun getSavePosition(): Int {
+        return selectedPosition
+    }
```

Comment:
We can make this look better.

Reply:
fun getSavedPosition(): Int = selectedPosition